src/lib/object-class.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Returns the object class for any value.
+ *
+ * @param {*} [value] - A value whose object class should be determined.
+ *
+ * @example
+ * const isNull = objectClass(null) === 'null'
+ * const isPOJO = objectClass({}) === 'object'
+ *
+ * @returns {String}
+ */
+const objectClass = (value) => /\[\w+ (\w+)]/.exec(Object.prototype.toString.call(value))[1].toLowerCase()
```

Comment:
I can sympathize with that, though, this is the correct technical definition of what `Object.prototype.toString` is for in this use case. Per MND's docs for `toString`, under the heading ["Using toString() to detect **object class**"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/toString#Using_toString()_to_detect_object_class):

>toString() can be used with every object and allows you to get its class.

I'm not opposed to other names, but I think this is the most accurate one.